install_matlab_runtime.yml

```
@@ -0,0 +1,50 @@
+- name: Install Matlab Runtime
+  hosts:
+    - localhost
+  gather_facts: false
+  tasks:
+    - name: Update repository cache
+      become: Yes
+      apt:
+        update_cache: true
+        #upgrade: dist
+        cache_valid_time: 3600
+        force_apt_get: true
+
+    - name: Download matlab runtime
+      get_url:
+        url: https://ssd.mathworks.com/supportfiles/downloads/R2021a/Release/6/deployment_files/installer/complete/glnxa64/MATLAB_Runtime_R2021a_Update_6_glnxa64.zip
+        dest: ~/data/volume_2
+        mode: 0550
+
+    - name: Install libs for libxt.so.6 issue
+      # https://nl.mathworks.com/matlabcentral/answers/102840-why-do-i-get-a-java-error-libxt-so-6-cannot-open-shared-object-file-no-such-file-or-directory-wh
+      become: yes
+      apt:
+        force_apt_get: yes
+        name: "{{ packages }}"
+        allow_unauthenticated: true
+      vars:
+        packages:
+          - libx11-dev
+          - libxcursor1
+          - libxi6
+
+    - name: Install zip
+      become: yes
+      apt:
+        force_apt_get: yes
+        name: "{{ packages }}"
+        allow_unauthenticated: true
+      vars:
+        packages:
+          - zip
+
+    - name: Unzip the file
+      shell: unzip ~/data/volume_2/MATLAB_Runtime_R2021a_Update_6_glnxa64.zip -d ~/data/volume_2/MATLAB_Runtime
```

Be consistent with path
```suggestion
      shell: unzip /data/volume_2/MATLAB_Runtime_R2021a_Update_6_glnxa64.zip -d /data/volume_2/MATLAB_Runtime
```